packages/ncids-js/src/components/usa-footer/utils/footer-back-to-top.component.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * NCI Back To Top
+ */
+export class NCIBackToTop {
+	/** Default Element */
+	protected element: HTMLElement;
+	/** Callback for handling the toggle functionality.  */
+	private scrollListener: EventListener = () => this.handleScroll();
+
+	public constructor(element: HTMLElement) {
+		this.element = element;
+		this.initialize();
+	}
+
+	private handleScroll(): void {
+		const y = window.scrollY;
+		if (y > 0) {
+			this.element.classList.remove('hide');
+			this.element.classList.add('show');
+		} else {
+			this.element.classList.remove('show');
+			this.element.classList.add('hide');
+		}
+	}
+
+	/**
+	 * Resets component to a clean state.
+	 * @public
+	 */
+	public unregister(): void {
+		this.element.classList.remove('show', 'hide', 'nci-footer__return-to-top');
```

Comment:
see note in scss file about naming

```suggestion
		this.element.classList.remove('show', 'hide', 'usa-footer__nci-return-to-top');
```